src/test/java/redis/clients/jedis/tests/commands/ControlCommandsTest.java

```diff
@@ -47,6 +47,17 @@ public void bgsave() {
     }
   }
 
+  @Test
+  public void bgsave_schedule() {
```

Comment:
Let's keep it Java (as much as we can).
```suggestion
  public void bgsaveSchedule() {
```